arch/arm/boot/dts/overlays/README

```diff
@@ -622,6 +622,30 @@ Params: gpiopin                 Gpio pin connected to the INTA output of the
         addr                    I2C address of the MCP23017 (default: 0x20)
 
 
+Name:   mcp23s17
+Info:   Configures the MCP23S08/17 SPI GPIO expanders.
+        If devices are present on SPI1 or SPI2, those interfaces must be enabled
+        with one of the spi1-1/2/3cs and/or spi2-1/2/3cs overlays.
+        If interrupts are enabled for a device on a given CS# on a SPI bus, that
+        device must be the only one present on that SPI bus/CS#.
+Load:   dtoverlay=mcp23s17,<param>=<val>
+Params: mcp23s08-spi<n>-<m>-present  4-bit interger, bitmap indicating MCP23S08
+                                     devices present on SPI<n>, CS#<m>
+
+        mcp23s17-spi<n>-<m>-present  8-bit interger, bitmap indicating MCP23S17
```

Comment:
These identifiers are very long, and that has caused you to change the indentation on the descriptions, which is not allowed. You might want to consider shortening some of the names, but if not then I suggest you start the descriptions on the following lines.